TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -446,7 +446,7 @@ private void WritePSMResults()
 
             // write PSMs
             string writtenFile = Path.Combine(Parameters.OutputFolder, "AllPSMs.psmtsv");
-            WritePsmsToTsv(Parameters.AllPsms, writtenFile, Parameters.SearchParameters.ModsToWriteSelection, CommonParameters.QValueOutputFilter);
+            WritePsmsToTsv(filterPSMList(Parameters, Parameters.AllPsms), writtenFile, Parameters.SearchParameters.ModsToWriteSelection);
```

Comment:
call FilterPsmList once at the top of WritePsmResults and then just use that list for each of the outputs, instead of calling the method 4x